src/libraries/System.Private.Xml/src/System/Xml/Serialization/XmlSerializationGeneratedCode.cs

```diff
@@ -327,7 +328,7 @@ private void GenerateTypedSerializers(Hashtable serializers)
                 _writer.Write("_tmp.Add(");
                 WriteQuotedCSharpString(key);
                 _writer.Write(", new ");
-                _writer.Write((string)serializers[key]);
+                _writer.Write((string?)serializers[key]);
```

Comment:
Another place where I'm curious if the code-gen has changed?

Reply:
no, I don't think this affects code-gen, why do you think so?